app/routes/dive_shop.py

```diff
@@ -0,0 +1,80 @@
+from flask import Blueprint, request
+from app.models import DiveShop
+from app import db
+from flask_jwt_extended import jwt_required, get_current_user
+
+bp = Blueprint('dive_shop', __name__, url_prefix="/shop")
```

Comment:
sorry i should've thought of this on the first review, but since this prefix is shop, maybe we should keep the file and blueprint name as shop and shop.py also, just so that it's all consistent